src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -4803,7 +4803,7 @@ public static Object toPrimitive(final Object array) {
         }
         final boolean[] result = new boolean[array.length];
         for (int i = 0; i < array.length; i++) {
-            result[i] = array[i].booleanValue();
+            result[i] = array[i];
```

Comment:
This is included here because it is directly invoked from `BooleanUtils`. There are other similar redundant unboxings in this class in addition to this.